apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.html

```diff
@@ -96,21 +96,45 @@ <h2>Icons</h2>
 
   <h2>Icon only</h2>
   <p>
-    The button can be rendered with an icon only and no text. This is useful for "close" buttons and
-    menu buttons.
+    The button can be rendered with an icon only and no visible text. This is useful for "close"
+    buttons and menu buttons.
+  </p>
+  <p>
+    To render a button with an icon only you can either include an accessible name inside the button
+    next to the icon and hide it visually by setting
+    <code>[showIconOnly]="true"</code>
+    or you can omit any text and set an
+    <code>aria-label</code>
+    instead. Please refer to the section on
+    <a href="#" (click)="scrollTo(accessibleIconButtons)">Accessible Icon Buttons</a>
+    below.
   </p>
-  <p>To render a button with an icon only, include an icon within the button and omit any text.</p>
   <p>
     <em>
-      Note: If you do need to include a text for the button in the markup but still want to render
-      the button as icon only, set
-      <code>[showIconOnly]="true"</code>
-      . This is also useful in scenarios where the button needs to toggle between the default state
-      and a "collapsed" state, e.g. when used in an
+      Note: including a—visually hidden—label in the markup is also useful in scenarios where the
+      button needs to toggle between the default state and a "collapsed" state, e.g. when used in an
       <a routerLink="../header">action group</a>
       .
     </em>
   </p>
+  <h3 #accessibleIconButtons>Accessible Icon Buttons</h3>
+  <p>
+    When rendering a button with no visible text it's important to make the button accessible to
+    assistive technologies, such as screen readers.
+  </p>
+  <p>
+    By including a—visually hidden—label in the markup as mentioned above, assistive technologies
```

Comment:
I think it would read a little better if in this second instance it is just stated plainly. But dealer's choice!
```suggestion
    By including a visually hidden label in the markup as mentioned above, assistive technologies
```